SU-SWS/stanford_bean_types
# SOE-1651: Add call to action modules

# NOT READY FOR REVIEW

# Summary
- This PR is for the call-to-action blocks. This is for the 2.x branch, and _SoE needs the 2.x branch_. 

# Needed By (Date)
ASAPP (As soon as practically possible)

# Urgency
We need to get stylin'

# Steps to Test
 
## To test:
1. Enable "Stanford Bean Call-to-Action" and "Stanford Bean  Call-to-Action Administration" module
2. Revert feature if necessary
3. On the "Manage Display" form adjust fields as necessary. (See Attachments)
4. Create a  **Call-to-Action** Bean
5. Verify the help text is displaying properly. See [SOE-1864](https://stanfordits.atlassian.net/browse/SOE-1864) and [SOE-1865](https://stanfordits.atlassian.net/browse/SOE-1865)
6.  Navigate to **admin/manage/content** and verify it displays properly in the mange content view
7. Click on **Manage Call-to-Action Blocks** menu item
8. Verify it shows up on the Manage content page and that the page provides expected options
9. Place block and verify display and responsive behavior

# Affected Projects or Products
SoE

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-1864
- https://stanfordits.atlassian.net/browse/SOE-1865
- https://stanfordits.atlassian.net/browse/SOE-1651

- Anyone who should be notified? @boznik, @josephgknox 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

# Attachments
Here's how to adjust the _Manage Display_

![screen shot 2017-06-20 at 11 09 03](https://user-images.githubusercontent.com/284440/27348628-9874d77e-55a9-11e7-9dfc-a192455854a2.png)
